pkg/qdr/qdr.go

```diff
@@ -482,7 +493,7 @@ type TcpEndpoint struct {
 	Address        string `json:"address,omitempty"`
 	SiteId         string `json:"siteId,omitempty"`
 	SslProfile     string `json:"sslProfile,omitempty"`
-	VerifyHostname *bool  `json:"verifyHostname,omitempty"`
+	VerifyHostname *bool  `json:"verifyHostname"`
```

Comment:
Since TcpEndpoint is used for both tcpListener and tcpConnector, I believe that `omitempty` flag must
be preserved as the tcpListener does not have the `verifyHostname` property defined.

Reply:
I see. The problem I noticed is that when I create a Listener, the respective tcpListener rendered
into the router config, has the following value, which is invalid and seems to be causing problems to the router:

```
                "verifyHostname": null
```

Reply:
Fixed